.github/workflows/release-vsix.yml

```
@@ -68,29 +80,114 @@ jobs:
+              }
+            ]
+          }
+          EOF
+          )
+          curl -X POST \
+            -H 'Content-Type: application/json' \
+            "${{ secrets.EDITOR_TEAM_CHAT_API }}&messageReplyOption=REPLY_MESSAGE_FALLBACK_TO_NEW_THREAD" \
+            -d "$body"
 
       - name: Create a release in wso2/ballerina-vscode repo
         uses: ./.github/actions/release
```

actionlint (1.7.12)

[error] description is required in metadata of "Release vsix to github and notify" action at "/home/jailuser/git/.github/actions/release/action.yml"

(action)